lang/en/lang.php

```diff
@@ -58,3 +58,8 @@
 $lang['preparingdownload']      = "We are preparing your download, please wait...";
 $lang['titleerrordownload']     = "Error";
 $lang['faileddownload']         = "There was a problem generating your download.";
+
+$lang['select_namespace']       = "Add Namespace to Selection";
```

Comment:
"Add a Namespace…" is my favourite.

I see that updating that Clear button is not that complicated. Probably I will do it, but that is then not in this PR. To prevent double work my suggestion is to use now a link for this instead of button.